core/tests/protractor_utils/AdminPage.js

```diff
@@ -195,8 +193,9 @@ var AdminPage = function() {
       'Starting one off jobs taking too long to appear.');
     var text = await (await oneOffJobRows.get(i)).getText();
     if (text.toLowerCase().startsWith(jobName.toLowerCase())) {
-      await (await oneOffJobRows.get(i)).element(
-        by.css('.protractor-test-one-off-jobs-start-btn')).click();
+      var oneOffJobRowsButton = (await oneOffJobRows.get(i)).element(
+        by.css('.protractor-test-one-off-jobs-start-btn'));
```

Comment:
This isn't something I expect you to know yet, but you don't actually need an `await` here. `.get(i)` returns an `ElementFinder`, not a `Promise`, so `await` doesn't actually do anything. This can be just:

```suggestion
      var oneOffJobRowsButton = oneOffJobRows.get(i).element(
        by.css('.protractor-test-one-off-jobs-start-btn'));
```